source/includes/changelogs/releases/100.7.1.rst

```diff
@@ -0,0 +1,27 @@
+.. _100.7.1-changelog:
+
+100.7.1 Changelog
+-----------------
+
+*Released 2023-05-24*
+
+This release fixes a few bugs and adds downloads for macOS 11 on ARM
+as well as RedHat Enterprise Linux 9 (x86 and ARM) and Amazon Linux
+2023 (x86 and ARM).
```

Comment:
The "as well as" and "and" in this sentence make this a little hard to parse in terms of which binary applies to which platform. What if we did something like:

This release fixes a few bugs and adds downloads for the following platforms:

- macOS 11 on ARM
- RedHat Enterprise Linux 9 (x86 and ARM)
- Amazon Linux 2023 (x86 and ARM)